screenpy/narration/narrator.py

```diff
@@ -66,12 +69,23 @@ def _chainify(narrations: List[BackedUpNarration]) -> ChainedNarrations:
 class Narrator:
     """The narrator conveys the story to the audience."""
 
-    def __init__(self, adapters: Optional[List[Adapter]] = None) -> None:
-        self.adapters: List[Adapter] = adapters or []
-        self.on_air = True
-        self.backed_up_narrations: List[List[BackedUpNarration]] = []
-        self.exit_level = 1
-        self.handled_exception = None
+    _instance = None
+
+    def __new__(
+        cls: Type[SelfNarrator], adapters: Optional[List[Adapter]] = None
+    ) -> SelfNarrator:
+        if cls._instance is None:
+            cls._instance = super().__new__(cls)
+            cls.adapters: List[Adapter] = []
+            cls.on_air = True
+            cls.backed_up_narrations: List[List[BackedUpNarration]] = []
+            cls.exit_level = 1
+            cls.handled_exception = None
+
+        if adapters is not None:
+            cls._instance.adapters = adapters
```

Comment:
So, this is one of the cool things, i think.

If you call `Narrator` with some new adapters, all the narration will use those new adapters! If you call `Narrator()`, it won't clear out the adapters. But if you call `Narrator(adapters=[])`, it will! 

I dunno, i think that might make it easier to use `Narrator` for _someone_. I mostly just handled it this way because i wanted to get the Narrator through `Narrator()`.

Reply:
I do this in conftest.py
```python
the_narrator.adapters = [StdOutAdapter(StdOutManager(create_logger()))]
```
Which would turn into this after?
```python
Narrator(adapters=[StdOutAdapter(StdOutManager(create_logger()))])
```

Reply:
You can choose whether to leave it as is or update it to the other one, both will still work!